app/default/data.py

```diff
@@ -103,3 +103,20 @@ def get_round_data(fund_id, round_id, as_dict=False):
         return Round.from_dict(round_response)
     else:
         return round_response
+
+
+def get_round_data_fail_gracefully(fund_id, round_id):
+    try:
+        round_request_url = Config.GET_ROUND_DATA_FOR_FUND_ENDPOINT.format(
+            fund_id=fund_id, round_id=round_id
+        )
+        round_response = get_data(round_request_url)
+        current_app.logger.error(f"{ round_response }")
```

Comment:
It is an error if we make a call to the fund store and it fails - we're logging as an error because it should never fail - we are tolerant of its failure, but if its down we need to know.

Reply:
but isn't the failure condition what's in the `except` block below? This line looks like part of the 'it worked' path?

Reply:
Aye I was looking at the wrong line your right I've removed .